app/models/manageiq/providers/ansible_tower/provider.rb

```diff
@@ -62,7 +62,14 @@ def self.process_tasks(options)
   end
 
   def self.create_audit_event(options)
-    msg = "'#{options[:task]}' initiated for #{options[:ids].length} #{ui_lookup(:table => 'providers').pluralize}"
+    msg = "'%{task}' initiated for %{amount} %{providers}" % {
+      :task      => options[:task],
+      :amount    => options[:ids].length,
+      :providers => Dictionary.gettext('providers',
+                                       :type      => :table,
+                                       :notfound  => :titleize,
+                                       :plural    => true,
```

Comment:
shouldn't this be `:plural => (options[:ids].length > 1)`, so that it is plural when it needs to be?